installer/PowerToysBootstrapper/bootstrapper/bootstrapper.cpp

```diff
@@ -42,7 +42,9 @@ std::optional<fs::path> ExtractEmbeddedInstaller(const fs::path extractPath)
         return std::nullopt;
     }
 
-    auto installerPath = extractPath / L"PowerToysBootstrappedInstaller-" PRODUCT_VERSION_STRING L".msi";
+    std::wstring msiName(L"PowerToysSetup-" STRINGIZE(VERSION_MAJOR) "." STRINGIZE(VERSION_MINOR) "." STRINGIZE(VERSION_REVISION) L"-");
+    msiName += get_architecture_string(get_current_architecture()) + std::wstring(L".msi");
```

Comment:
could you please add TODO, since when we have arm builds, we'll need to get the architecture string at build time instead of runtime, since we could e.g. launch x64 installer on arm system etc.

Reply:
There is already a TODO in get_current_architecture method.